js/src/components/app-select-control/index.js

```diff
@@ -16,14 +16,29 @@ import './index.scss';
  * it will be added to the container div's `className`,
  * so that you can further control its style.
  *
- * @param {*} props
+ * @param {*} props The component props.
+ * @param {boolean} [props.selectSingleValue=false] Whether the select should show only one value.
  */
 const AppSelectControl = ( props ) => {
-	const { className, ...rest } = props;
+	const { className, options, selectSingleValue, ...rest } = props;
```

Comment:
Need to set the default value for `selectSingleValue` here.
```suggestion
	const { className, options, selectSingleValue = false, ...rest } = props;
```